selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/CheSeleniumSuiteModule.java

```diff
@@ -100,14 +102,17 @@ public void configure() {
     bind(TestWorkspaceAgentApiEndpointUrlProvider.class)
         .to(CheTestWorkspaceAgentApiEndpointUrlProvider.class);
 
-    bind(TestWorkspaceProvider.class).to(TestWorkspaceProviderImpl.class).asEagerSingleton();
+    bind(TestWorkspaceProvider.class).to(CheTestWorkspaceProvider.class).asEagerSingleton();
     bind(TestWorkspaceUrlResolver.class).to(CheTestWorkspaceUrlResolver.class);
 
+    bind(TestWorkspaceServiceClient.class).to(CheTestWorkspaceServiceClient.class);
+
     install(new FactoryModuleBuilder().build(TestUserHttpJsonRequestFactoryCreator.class));
-    install(new FactoryModuleBuilder().build(TestWorkspaceServiceClientFactory.class));
+    install(new FactoryModuleBuilder().build(CheTestWorkspaceServiceClientFactory.class));
```

Comment:
We shouldn't use `CheTestWorkspaceServiceClientFactory` here.

Reply:
Could you please shortly elaborate why?

Reply:
Because `TestWorkspaceServiceClientFactory` should work perfectly well with `CheWorkspaceServiceClient` itself.